eq-author/src/components/PreviewSectionRoute/__snapshots__/SectionIntroPreview.test.js.snap

```diff
@@ -11,25 +11,26 @@ exports[`SectionIntroPreview should render an error when the content is missing
       }
     />
   </SectionIntroPreview__TitleBlock>
-  <Error
-    data-test="no-content"
-    large={true}
-    margin={false}
-  >
-    Missing Introduction Content
-  </Error>
+  <SectionIntroPreview__ContentBlock
+    dangerouslySetInnerHTML={
+      Object {
+        "__html": "<h2></h2>",
```

Comment:
That's an empty tag so I guess the test needs to be updated as it looks like it was checking for an empty intro and `<h2></h2>` was empty before.

Reply:
I have updated the snapshots so they reflect the new empty state